### PR TITLE
Feature/send statistics

### PR DESCRIPTION
**Pull request by adelkahomolova**
## Description

Send details of security issues from yarn and npm audit.
Unify returned data from yarn and npm audit.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have updated the documentation accordingly.
- [ ] I have added new practice to practice list in README.md.
- [x] I have read the **CONTRIBUTING** document.
- [x] I haven't repeated the code. (DRY)
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/practices/PracticeUtils.spec.ts [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  describe('parseYarnAudit', () => {
+    it('All properties should be defined', async () => {
+      const result = await parseYarnAudit(yarnShellResponse as ShellString);
+      const isEmpty = !Object.values(result).some((x) => x !== null && x !== undefined);
+      expect(isEmpty).toBe(false);
+    });
+  });
+});
+
+const yarnShellResponse =
```
Comment (prokopsimek) [inline src/practices/PracticeUtils.spec.ts:54]: use `*.mock.ts`

File: src/reporters/DashboardReporter.ts [hunk 2]
```diff
@@ -49,6 +48,7 @@ export class DashboardReporter implements IReporter {
       id: uuid.v4(),
       dxScore: { value: dxScore.value, points: dxScore.points },
     };
+    const securityVulnerabilitiesPractice = practicesAndComponents.filter((p) => p.practice.id === 'JavaScript.SecurityVulnerabilities');
```
Comment (prokopsimek) [inline src/reporters/DashboardReporter.ts:51]:
```suggestion
    const securityVulnerabilitiesPractice = practicesAndComponents.find((p) => p.practice.id === 'JavaScript.SecurityVulnerabilities');
```

File: src/reporters/DashboardReporter.ts [hunk 3]
```diff
@@ -57,7 +57,7 @@ export class DashboardReporter implements IReporter {
       const componentWithScore: ComponentDto = {
         component: cwp.component,
         dxScore: { value: dxScoreForComponent, points: dxScorePoints },
-        securityIssues: [],
+        securityIssues: <SecurityIssueDto[]>securityVulnerabilitiesPractice[0]?.practice.data?.statistics?.securityIssues,
```
Comment (prokopsimek) [inline src/reporters/DashboardReporter.ts:60]:
```suggestion
        securityIssues: <SecurityIssueDto[]>securityVulnerabilitiesPractice?.practice.data?.statistics?.securityIssues,
```

File: src/reporters/DashboardReporter.ts [hunk 4]
```diff
@@ -87,10 +87,21 @@ export type DxScoreDto = Pick<DXScoreResult, 'value' | 'points'>;
 //security issues
 export type SecurityIssueDto = {
   library: string;
-  currentVersion: string;
   type: string;
-  newestVersion: string;
   severity: SecurityIssueSeverity;
+  vulnerable_versions: string;
```
Comment (prokopsimek) [inline src/reporters/DashboardReporter.ts:92]:
```suggestion
  vulnerableVersions: string;
```

File: src/reporters/DashboardReporter.ts (hunk 3 above)
Comment (adelkahomolova) [reply src/reporters/DashboardReporter.ts:60]: Why? Method filter is returning array - type `PracticeWithContextForReporter[]`

Comment (prokopsimek) [reply src/reporters/DashboardReporter.ts:60]: Because `find` already returns only one

**Pull request merged**